package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Vue.js create organization wizard component for Bedrock Web apps",
   "main": "index.js",
   "dependencies": {
-    "bedrock-vue-address-form": "digitalbazaar/bedrock-vue-address-form#quasar-v1",
+    "bedrock-vue-address-form": "github:digitalbazaar/bedrock-vue-address-form#quasar-v1",
```

Comment:
Is this change needed?

Reply:
no, but keeps it consistent with other package.json files and is not necessary.

Reply:
@gannan08 let me know if this is good to go.